db/schema.rb

```diff
@@ -59,7 +59,7 @@
   end
 
   create_table "transactions", force: :cascade do |t|
-    t.integer "credit_card_number"
+    t.string "credit_card_number"
```

Comment:
Why is it better to have as a string compared to an integer?

Reply:
In the database you can only store integers of a certain size and the CC is to large. Realistically the CC would be encrypted so we would be storing it differently anyway. But, this is the only way to store it for now.

Reply:
Makes sense, thanks!